### DISCUSSION TITLE
Wormhole WRENBTC sent from Solana to RENBTC on Ethereum - Missing

### DISCUSSION THREAD

**Discussion by tosunra** [wormhole-foundation/wormhole#384] [Problems / bugs] [answered]
Hello , i transfer wrenbtc from Solana to Eth( wormhole) , i refresh the page, and the wrenbtc was sent, but i can not get them in my eth wallet. It just show In Process, how can i do now? Please help.

https://explorer.solana.com/tx/L67137mcs84KefCyH9WRauCG88VwH9fKFHWJrQ4WnQF2gKeDhx7ZYTR4R1cWjmX4nRF9i8cYSv94mNGqaHuZBEj

**Comment by leoluk** [comment, marked as answer]
The transaction was successfully processed by the Wormhole network, but the client-side transaction submission may have failed. Try clearing your cache and reloading wormholebridge.com to see if the transaction can be completed there. Make sure both wallets are connected.

If wormholebridge.com doesn't work, you can manually submit the signed VAA bytes below to the submitVAA method on the Wormhole contract: https://etherscan.io/address/0xf92cd566ea4864356c5491c177a430c222d7e678#writeContract

---

Digest (for reference only): f353126bc0fde9605cf2d60dd2efec89964f021f8645ce27e28cf0b1b6cc091f

Signed VAA bytes to submit to the contract:

> 0x01000000010d00e03abeacb7de8d8ac6064b623d09fc0df121544c93cb84ce81031b023b4c330d5594c09e55dadecc21e35d8078a08b92b0c712a3344efb61be18a622fd8329660101b261f04a5f495875bd6ffc845b8bb78bdc82a1aa9a1bd7b1e7d6247eaf0571ff4f5b8f097a2afba2896b24776282051d06b6a285b8275020a955715bce298af10002f9b75b2161220f2e245e801c88765ff83bb8e5f43280b78f96a05b95af5ed2fa36c3a6c18487c3af73abc2799d37b1bc13fab1aabc3c36a85a88162fc87d8e9a0003fe21b9634a5d64bebd1eedbe5fac786092f0acb15cc9ddd7e44fdf578dd0424d1d966c536aab6e289190542143bf26476dee8d2a7224695e70c0345d59a3136901047a817f8afb9030e7dbdb5aabcfc5aad2ce5ada99488f3302a61d97e3d2dc5eea3db5b6d253a6404df7a08770ca9cb5c3e2a1867e4de6781891bb9765d21c5319000563eb7b713299720605afaabd681737ec96ec68fbe839a57040694e8814af215653feb7b4bd959d0b87a34c092883a7565d5025c34a51c063084f0184049ec9bc0007dd957f2d13e51fc134c1c414825b20ae44f1120b5ea2ebabdeec2ebd4a541c46707d1502e33302bd71b4250d0391f246a3102bc190409942c9cb6e990aeb7b9c0108d1f638d1ce71b805b713ca1a17930ef714a098eb0126a53ff5c9a5ad22f9f7500812560e5e255f3c43c13494a0bf89760d1b4a80e707a9fda4037b7d37382856000af1c0c560ca3b6d210aa8665283ece0072ff2fe574312ae3561bbc450a62c14801bd79909be62db88a9fd6dc47e06ed7eef6baee317b87fb6efc1dd5ed37dddea010d23f97268dd76ab23636287214a6cef13b9ecf2b74b09e50dcbf81369ec0a964977f0485549aad6ff1c3bd2e237cce006a48dd5d50b78280fafc13d47656cf3ec01109f0d42da7d6511f50442ffe3d1dc0082a57f3144dcf32714207b5902f4b5f32365a52dedfaf536a9c09a048ba52f483af70e202ef9a95844a8cdc0653ac42a750011332830c0ac88d97e25bc9b69e120d9a641a777c207a0ee15af9e4b3dfd59684d25430ef33846d24fc58525fde4915374adaa719fbc635fc250f08627064bfcaf01128fb01c190d8715e7b40b9d4edbb730f483a293726a04efbc3518866d9a7f54d8437ccfb9d3d451b7ac82c8d4e92e0d91fd8b738b1bfb075c8770132a33a749540061352a2c100000924701022814166eb5c0480186dd6e06df78b51aea226431d9f760bab15d170d9e20d5700000000000000000000000004a7b0acc6b89fc01185b38b77c0a53fd85230ff202000000000000000000000000eb4c2781e4eba804ce9a9803c67d0893436bb27d0800000000000000000000000000000000000000000000000000000000844ccc21
